obstacle_stop_planner_refine/design/design.md

```diff
@@ -0,0 +1,11 @@
+class ObstacleStopPlannerNode
+
+Key input: pathCallback(input/trajectory)
+
+Class diagram
+
+![img](img/class_diagram_orig.png)
+
+Proposal class diagram
+
+![img](img/class_diagram_refine-1.png)
```

Comment:
I think this information should be written in this PR, not need to manage with git.

Reply:
Thanks. I removed the file.

Reply:
@KeisukeShima Could you leave the images as comments in this PR? It's very useful!